homeassistant/components/switchbot_cloud/__init__.py

```diff
@@ -109,6 +129,50 @@ async def async_setup_entry(hass: HomeAssistant, config: ConfigEntry) -> bool:
     await gather(
         *[coordinator.async_refresh() for coordinator in coordinators_by_id.values()]
     )
+
+    if any(
+        coordinator.update_by_webhook() for coordinator in coordinators_by_id.values()
+    ):
+        # Need webhook
+
+        # Get/create config to store a unique id for this hass instance.
+        store = Store[dict[str, Any]](
+            hass, STORAGE_VERSION, STORAGE_KEY + "-" + config.entry_id
+        )
+        if not (persistent_config := await store.async_load()):
+            # Create config
+            persistent_config = {
+                CONF_WEBHOOK_ID: secrets.token_hex(),
```

Comment:
Why are we storing this in a store instead of the config entry data?

Reply:
Hum, I don't known.  
I'm new in ha development (but I'm a experienced developer in other side).  
So, I'm inspired in other integration like samsung smartthing but if there is a best way, lest's go

Reply:
Withings is a good one to check

Reply:
ok, I'll check it that  
many thanks